### DISCUSSION TITLE
Mac OS Monterey

### DISCUSSION THREAD

**Discussion by jackdellis** [elrumo/macOS_Big_Sur_icons_replacements#1366] [General]
Hello, 

I'm trying to change some of my app icons, on Monterey it doesn't seem to work with the drag and drop, I am going into applications clicking get info on the app, then dragging my icns file on top of the icon at the top left of the window. Usually this will change the app icon. It isn't working for any of the native apps such as Notes or App Store, it did work for a program that I had downloaded. 

Is anyone else having this issue and is there a workaround?

Thanks

**Comment by OwenMille** [comment]
I was able to do this on monterey + apple silicon chip by disabling SIP.  You can just look at some articles online that talk about changing the finder icon, it is the same process for some of these default applications.
https://www.igeeksblog.com/how-to-change-finder-dock-icon-on-mac/

**Comment by Astrofive** [comment]
I had the same issue and realized that you have to drag the icon file directly onto this specific area, circled in red.

<img width="1199" alt="Screen Shot 2022-04-07 at 7 39 17 PM" src="https://user-images.githubusercontent.com/6345977/162352646-ae97cfe6-8f70-456b-8896-adedfa9bc65a.png">
